brainstormforce/astra
# AST-2270 - Accessibility option

### Description
Jira : https://brainstormforce.atlassian.net/browse/AST-2270

### Screenshots
https://d.pr/i/UpUHWU

### Types of changes
New feature (non-breaking change which adds functionality) 

### How has this been tested?
-Check blog and single page 
- Check in frontend 
- Check if all the options added worked properly

### Checklist:


[AST-2270]: https://brainstormforce.atlassian.net/browse/AST-2270?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
